client/src/core/client/admin/components/ModerateCard/CommentRevisionContainer.tsx

```diff
@@ -100,6 +107,10 @@ const enhanced = withFragmentContainer<Props>({
           ... on TenorMedia {
             url
             title
+            width
+            height
+            tenorStill: still
+            tenorVideo: video
```

Comment:
If we're aliasing these to `still` and `video` when we consume them from the API client side, do we want to call them `TenorMedia > tenorStill` and `TenorMedia > tenorVideo`? Seems like a duplication of the naming?

Reply:
Hmm, I was trying to keep this consistent with the naming for `GiphyMedia`, except that we might not have a `still` or `video` for some Tenor gifs, so types are different.